### DISCUSSION TITLE
Telemetry blocking

### DISCUSSION THREAD

**Discussion by naumgod** [tnodir/fort#48] [Ideas]
Please add a setting to automatically block telemetry in Windows 11. I don’t like using any tweakers because then Windows doesn’t update, something else doesn’t work.

**Comment by tnodir** [comment]
You can use [WindowsSpyBlocker](https://github.com/crazy-max/WindowsSpyBlocker) IP addresses list:

1. Create new Zone:
![zone](https://user-images.githubusercontent.com/303655/218735681-b67b730b-12d4-4ffc-b668-c0b6bc4e6625.jpg)

2. Schedule the Zones Downloader:
![scheduler](https://user-images.githubusercontent.com/303655/218735963-4f16fda9-8202-4ae9-ac8b-0cc54dac2453.jpg)

3. Exclude the Zone from Allowed Internet Addresses:
![exclude-zone](https://user-images.githubusercontent.com/303655/218736419-53ddc9e0-2911-4046-8d4e-0720dc8ecb97.jpg)

**Comment by naumgod** [comment]
where do they get telemetry ip addresses from? I understand that the list can be added to the host file. But I know and I did that in * .exe you can add a back entrance and you can enter. For example xbox, netflix, or some kind of explorer.
You can add what would automatically be the addition of filters, as users of TV owners do, so that the IP playlist is automatically updated.

**Reply by tnodir** [reply]
Не понял вопроса. Можешь на русском написать?

**Comment by Tarek-Hasan** [comment]
It would be nice if you include [WindowsSpyBlocker](https://github.com/crazy-max/WindowsSpyBlocker) in Fort by default, same as [SimpleWall](https://github.com/henrypp/simplewall) did.

**Reply by tnodir** [reply]
@Tarek-Hasan Maybe, sometimes in the future, there will be some preset block-lists to enable them by one click (as in Simplewall).

But now you have to [configure it by hand](https://github.com/tnodir/fort/discussions/48#discussioncomment-4970197).

**Comment by DWolf19** [comment]
[WindowsSpyBlocker](https://github.com/crazy-max/WindowsSpyBlocker) The project has been stale but it will be back, this can be the alternative https://github.com/undergroundwires/privacy.sexy
https://github.com/crazy-max/WindowsSpyBlocker/issues/514#issuecomment-2029685751
